src/main/java/sirius/kernel/cache/CacheRemoverBuilder.java

```diff
@@ -0,0 +1,15 @@
+package sirius.kernel.cache;
+
+import java.util.function.BiFunction;
+import java.util.function.BiPredicate;
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public interface CacheRemoverBuilder<K, V, T> {
+    <R> CacheRemoverBuilder<K, V, R> map(BiFunction<String, T, R> mapper);
```

Comment:
probably not needed - as far as i understand it right now...

Reply:
Its very little implementation overhead, and it does allow some more flexibility.